docs/coordinates/matchsep.rst

```diff
@@ -249,8 +249,9 @@ the catalog:
 
 .. doctest-requires:: scipy
 
+    >>> import numpy as np
     >>> matches = catalog[idx]
-    >>> (matches.separation_3d(c) == d3d).all()
+    >>> np.allclose(matches.separation_3d(c), d3d, rtol=2e-16, atol=3e-13)
```

Comment:
This is user-facing documentation, so having to import `numpy` and specify `rtol` and `atol` is distracting. It would be much better to display what `d3d` is when it is computed, then display `matches.separation_3d(c)` and let the reader compare the results themselves. Then we could also let the [`+FLOAT_CMP` directive](https://github.com/scientific-python/pytest-doctestplus?tab=readme-ov-file#floating-point-comparison) take care of any floating point rounding differences in a way that is invisible to the readers.

Ideally the entire section would be rewritten in a similar manner, but it might be best to make a minimal change here just to solve the immediate problems in CI.

Reply:
Sounds reasonable. I agree that invoking `np.allclose` is distracting. I just pushed your suggestion !